group_example_soln.py

```diff
@@ -0,0 +1,86 @@
+group = {
+    "Jill": {
+        "age": 26,
+        "job": "biologist",
+        "relations": {
+            "Zalika": "friend",
+            "John": "partner"
+        }
+    },
+    "Zalika": {
+        "age": 28,
+        "job": "artist",
+        "relations": {
+            "Jill": "friend"
+        }
+    },
+    "John": {
+        "age": 27,
+        "job": "writer",
+        "relations": {
+            "Jill": "partner"
+        }
+    },
+    "Nash": {
+        "age": 34,
+        "job": "chef",
+        "relations": {
+            "John": "cousin",
+            "Zalika": "landlord"
+        }
+    }
+}
+
+def max_age(group):
+    """ Finds the maximum age of a group of people.
+    """
+    ages = []
+    for i in range(len(group)):
+        vals = list(group.values())[i]
+        ages.append(vals["age"])
+    return max(ages)
```

Comment:
This certainly does work, can you do them all with comprehension expressions?

So something like this
```suggestion
    ages = [person["age"] for person in group.values()]
    return max(ages)
```